roles/edpm_kernel/tasks/kernelargs.yml

```diff
@@ -100,13 +100,25 @@
         - /boot/efi/EFI/fedora
 
     - name: Generate grub config
-      ansible.builtin.command: "grub2-mkconfig -o /boot/grub2/grub.cfg"
+      ansible.builtin.command: >-
+        grub2-mkconfig -o /boot/grub2/grub.cfg
+        {{ '--update-bls-cmdline'
+        if ansible_distribution == 'RedHat'
+        and ansible_distribution_major_version | int >= 9
+        and ansible_distribution_minor_version | int >= 3
```

Comment:
Does  `ansible_distribution_minor_version` var even exist? I can only see the following facts. Also, it's not good to use vars and instead use `ansible_facts['distribution_major_version']` and we should disable `inject_facts_as_vars` which is very expensive.

`       
        "distribution": "RedHat",
        "distribution_file_parsed": true,
        "distribution_file_path": "/etc/redhat-release",
        "distribution_file_search_string": "Red Hat",
        "distribution_file_variety": "RedHat",
        "distribution_major_version": "9",
        "distribution_release": "Plow",
        "distribution_version": "9.4",
`

Reply:
My bad I was using very old docs. Should be fixed now.